packages/modular-scripts/src/buildPackage/makeBundle.ts

```diff
@@ -46,8 +55,7 @@ export async function makeBundle(
     includePrivate,
   );
 
-  const packageJsonName = packageJson.name as string;
-  logger.log(`building ${packageJsonName} at ${packagePath}...`);
+  logger.log(`building ${target} at ${packagePath}...`);
```

Comment:
Don't need to include packagePath here because of the logger but maybe instead include the outputDirectory instead?